Libraries/VaahPayPal.php

```diff
@@ -0,0 +1,1014 @@
+<?php
+namespace WebReinvent\VaahExtend\Libraries;
+
+use Cartalyst\Stripe\Laravel\Facades\Stripe;
+use PayPal\Api\OpenIdUserinfo;
+
+
+use Illuminate\Http\Request;
+use PayPal\Api\Agreement;
+use PayPal\Api\Amount;
+use PayPal\Api\Item;
+use PayPal\Api\ItemList;
+use PayPal\Api\MerchantPreferences;
+use PayPal\Api\Patch;
+use PayPal\Api\PatchRequest;
+use PayPal\Api\Payer;
+use PayPal\Api\Payment;
+use PayPal\Api\PaymentDefinition;
+use PayPal\Api\PaymentExecution;
+use PayPal\Api\Plan;
+use PayPal\Api\RedirectUrls;
+use PayPal\Api\ShippingAddress;
+use PayPal\Api\Transaction;
+use PayPal\Auth\OAuthTokenCredential;
+use PayPal\Common\PayPalModel;
+use PayPal\Rest\ApiContext;
+
+class VaahPayPal{
+
+    private $apiContext;
+    private $mode;
+    private $client_id;
+    private $client_secret;
+    private $return_url;
+    private $cancel_url;
+
+    public function __construct()
+    {
+        $this->mode = env('PAYPAL_MODE');
+        $this->client_id = $this->mode == 'sandbox' ? env('PAYPAL_SANDBOX_CLIENT_ID') : env('PAYPAL_LIVE_CLIENT_ID');
+        $this->client_secret = $this->mode == 'sandbox' ? env('PAYPAL_SANDBOX_CLIENT_SECRET') : env('PAYPAL_LIVE_CLIENT_SECRET');
+        $this->return_url = url('/').'#/paypal/complete';
```

Comment:
@we-himanshu-r001-v1 is route should not be hard coded instead, you should accept a parameter in `constructor` with a default value like `url('/').'#/paypal/complete'`

Reply:
@we-himanshu-r001-v1 and route name should be a success

Reply:
Done Please refer [435246d](https://github.com/webreinvent/vaahextend/pull/6/commits/435246dc38610c6eae126af87b7daeb4c2bf9537)